src/Behat/Context/CookieComplianceContext.php

```diff
@@ -350,6 +350,21 @@ protected function showThirdPartyCookieUrls(array $urls) {
     $output->writeln("\nPlease check and exclude them if not applicable. Set the cookies_third_party_domains_ignored variable up.\n");
 
     echo $output->fetch();
+
+  /**
+   * Check cookie exists.
+   *
+   * @Given the cookie with name :cookie_name exists
+   * @Given the cookie with name :cookie_name exists with value :value
+   */
+  public function cookieExists($cookie_name, $value = NULL) {
```

Comment:
Is it okay for you to divide the method into two? One for checking it exists, another to check it has a specific value.

Reply:
@omarlopesino Done